Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/overview/PersonColorShade.kt

```diff
@@ -13,6 +13,7 @@ enum class PersonColorShade(
     @DrawableRes val maskIcon: Int = R.drawable.ic_mask,
     @DrawableRes val maskLargeBadgeBg: Int = R.drawable.mask_badge_bg,
     @DrawableRes val maskSmallBadge: Int = R.drawable.mask_small_badge,
+    @DrawableRes val noMaskSmallBadge: Int = R.drawable.no_mask_small_badge
```

Comment:
why do you need this new field? GREEN color is already overriding the value of maskSmallBadge to provide the right icon

Reply:
Nice, I didn't notice.